tinygrad/nn/__init__.py

```diff
@@ -209,6 +209,33 @@ def __call__(self, x:Tensor):
     # elementwise_affine on channels
     return x * self.weight.reshape(1, -1, *[1] * (len(x.shape)-2)) + self.bias.reshape(1, -1, *[1] * (len(x.shape)-2))
 
+class RMSNorm:
+  """
+  Applies Root Mean Square Layer Normalization over a summed inputs to a neuron in one layer.
+
+  - Described: https://paperswithcode.com/method/rmsnorm
+  - Paper: https://arxiv.org/pdf/1910.07467
+
+  ```python exec="true" source="above" session="tensor" result="python"
+  t = Tensor.rand(2, 5) * 2 + 1
+  rms_norm = nn.RMSNorm(t.size(-1), 1e-6)
+  print(t.std().item())
+  ```
+  ```python exec="true" source="above" session="tensor" result="python"
+  t = rms_norm(t)
```

Comment:
??

Reply:
I don't understand what's the problem here. Should rms_norm be renamed to norm?